WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -133,6 +133,11 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
 
         ABTest.start()
 
+        if FeatureFlag.useURLSession.enabled {
+            WordPressComRestApi.useURLSession = true
+            WordPressOrgXMLRPCApi.useURLSession = true
+        }
```

Comment:
Nitpick: I liked how the WooCommerce counterpart has this in a dedicated function. I feel like it's a great way to keep the configuration cod eat this level tidier.

Reply:
👍 Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22540/commits/1142472c0cb2b2a1b5acbba0e51a2512792aec8f